### PR TITLE
LGA 1831 assign face to face

### PR DESCRIPTION
**Pull request by Pheonnexx**
This is delivering a test which goes through to assign face to face contact.

### CODE REVIEW

File: behave/features/steps/assign_face_to_face.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from features.steps.common_steps import search_and_select_case, wait_until_page_is_loaded
+from features.steps.cla_in_scope import assert_header_on_page
+from selenium.webdriver.common.by import By
+
+
+@step(u'that I am on the Alternative Help page')
+def step_impl(context):
+    # The case used has been created for this test case.
+    # we select and then go through to alternative help
+    search_and_select_case(context, "AS-0001-0001")
```
Comment (psweeting1) [inline behave/features/steps/assign_face_to_face.py:10]: Might be a good idea to put `AS-0001-0001` into the constants.py file

File: behave/features/steps/assign_face_to_face.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    # The case used has been created for this test case.
+    # we select and then go through to alternative help
+    search_and_select_case(context, ASSIGN_F2F_CASE)
+    context.execute_steps(u'''
+        When I click on the Assign Alternative Help icon
+        Then I am taken to the "Alternative help" page for the case located at "/alternative_help/"
+    ''')
+
+
+@step(u'I am can select Assign F2F button')
```
Comment (jogilder) [inline behave/features/steps/assign_face_to_face.py:18]: Typo - I am can select

File: behave/features/steps/assign_face_to_face.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+def step_impl(context):
+    # This clicks the face to face link, which is hidden as a tab.
+    page = context.helperfunc
+    tabs = page.find_by_css_selector("ul.Tabs")
+    finance_tab_link = tabs.find_element_by_link_text("Face to Face")
```
Comment (jogilder) [inline behave/features/steps/assign_face_to_face.py:23]: name of variable should refer to face to face

File: behave/features/steps/assign_face_to_face.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    page.driver().execute_script("arguments[0].click();", finance_tab_link)
+    page.click_button(By.NAME, "assign-f2f")
+
+
+@step(u'I am taken to the call centre dashboard page')
```
[the remark this answers is not in the document]
Comment (Pheonnexx) [reply behave/features/steps/assign_face_to_face.py:28]: but this isnt navigating, this is checking that I have been automatically redirected to the call centre dashboard

[the remark this answers is not in the document]
Comment (jogilder) [reply behave/features/steps/assign_face_to_face.py:28]: Ok, but there is a step for that as well. `u'I am taken to the "{header}" page located on "{page}"'`

File: behave/features/steps/assign_face_to_face.py (hunk 2 above)
Comment (Pheonnexx) [reply behave/features/steps/assign_face_to_face.py]: fixed

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**